services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataFormView.jsx

```diff
@@ -483,18 +499,16 @@ const EmploymentDataFormView = (props) => {
                   <Select
                     showSearch
                     optionFilterProp="children"
-                    placeholder="choose classification"
+                    placeholder={<FormattedMessage id="admin.select" />}
```

Comment:
Should you be using a newly created id? If someone decides to change the one for admin, it might not work for this page anymore.

Reply:
will change :)